### ISSUE TITLE
Dual-licensing too vague (and possibly irrelevant)

### ISSUE THREAD

**Issue by jonassmedegaard** [at-import/toolkit#79]
Project is dual-licensed under MIT and GPL.

The MIT part is ambiguous - there are multiple licenses are labelled as "MIT", but that one is spelled out in the LICENSE file as being what unambiguously could be labelled as "Expat".

The GPL part is ambiguous too - and is not spelled out, just references the GNU web page for that license.  What is missing is which version, and if permitted to replace with newer versions or not.

...but technically dual-licensing should not be needed anyway, since Expat is compatible with GPLv2 and newer.

In short: Please consider either changing the "GPL" to e.g. "GPL-2+" (or the full paragraph recommended by GNU and listed at the bottom of their license page), or simply drop the GPL licensing as superfluous.

**Comment by jonassmedegaard** [comment]
Ah, I spotted now that Gemspec says "GPL-2.0" - so arguably this is all good (except that it is difficult to locate, and except it is still superfluous - e.g. if _only_ version 2 then that can be circumvented via the Expat licensing)

**Comment by scottkellum** [comment]
I just removed GPL. The idea was to support as many edge cases as possible meh, dual licence is always a bit confusing #yolo
